sdk/eventhub/Azure.Messaging.EventHubs.Processor/README.md

```diff
@@ -70,6 +74,9 @@ Once you have the connection strings, see [Creating an Event Processor Client](#
 
 For more concepts and deeper discussion, see: [Event Hubs Features](https://learn.microsoft.com/azure/event-hubs/event-hubs-features).
 
+### RBAC permissions
+v
```

Comment:
This looks like a mistake?

Reply:
uh... yup.   Changed the approach and forgot to delete the section.   Thanks!